modules/zeotapIdPlusIdSystem.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * This module adds Zeotap to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/zeotapIdPlusIdSystem
+ * @requires module:modules/userId
+ */
+import * as utils from '../src/utils.js'
+import {submodule} from '../src/hook.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const ZEOTAP_COOKIE_NAME = 'IDP';
+const storage = getStorageManager();
+
+function isValidConfig(configParams) {
+  if (!configParams) {
+    utils.logError('User ID - zeotapId submodule requires configParams');
+    return false;
+  }
+  if (!configParams.name) {
+    utils.logError('User ID - zeotapId submodule requires valid config params: name');
+    return false;
+  }
+  return true;
+}
```

Comment:
in your example you don't list any params, but you are requiring "name" here. what is the name param and how are you using it? i don't see it anywhere in your code

Reply:
This is removed, not required.